src/db_client/builder.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+use std::sync::Arc;
+
+use super::{cluster::ClusterImpl, standalone::StandaloneImpl, DbClient};
```

Comment:
We prefer not to use `super` outside test mod. Maybe other places in this crate have this as well.

I think we need a linter for this...

Reply:
Done.